jobs/scripts/nightly-samba-containers/nightly-samba-containers.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+CONTAINER_GIT_REPO="https://github.com/samba-in-kubernetes/samba-container"
+CONTAINER_GIT_BRANCH="main"
+CONTAINER_CMD=${CONTAINER_CMD:-podman}
+IMG_REGISTRY="quay.io"
+IMG_REGISTRY_BASE="${IMG_REGISTRY}/samba.org"
```

Comment:
I think this might as well be 1 variable. I don't think it will be very useful to only change the registry server but not the "repos path".

Also, do you plan on changing this to point at a non-canonical repo base for a period while we test and develop this feature? We touched on that in an earlier comment but I see it still using `quay.io/samba.org`.  We can use your quay.io account or mine. If you want to use mine I can create a unique bot account for this test and I will share the secret with you offline.

Reply:
> I think this might as well be 1 variable. I don't think it will be very useful to only change the registry server but not the "repos path".

I might have split it up for ease of use while authenticating to the registry with just `IMG_REGISTRY`.

> Also, do you plan on changing this to point at a non-canonical repo base for a period while we test and develop this feature? We touched on that in an earlier comment but I see it still using `quay.io/samba.org`. We can use your quay.io account or mine. If you want to use mine I can create a unique bot account for this test and I will share the secret with you offline.

Yes, please. I'll update to reflect yours(_quay.io/phlogistonjohn_) and wait for the bot account to be created.